src/lib/components/table/table.svelte

```diff
@@ -38,8 +48,8 @@
     {/each}
   </thead>
   <tbody>
-    {#each $table.getRowModel().rows as row}
-      <tr>
+    {#each $table.getRowModel().rows as row, index}
+      <tr on:click={() => onRowClick(index)} class:cursor-pointer={isRowClickable}>
```

Comment:
it really would be great if we could get these to be `<a>` elements in order to preserve navigability and allow people to "open in new tab" etc. But I understand that that might require a larger refactoring of the table component. Would probably be good to log in an issue for this so we don't lose track.

Reply:
yes agree, will make an issue